pcap_ex.c

```diff
@@ -290,7 +290,7 @@ pcap_ex_compile_nopcap(int snaplen, int dlt, struct bpf_program *fp, char *str,
 	char ebuf[PCAP_ERRBUF_SIZE];
 	int ret = -1;
 	
-	_mktemp(path);
+	mktemp(path);
```

Comment:
So why were the Windows-isms introduced by the "Warnings have been fixed. Timeout processing has been fixed for __next__."  commit?  Were those changes needed by some Windows compiler/library?